src/java/org/apache/nutch/net/URLFilters.java

```diff
@@ -41,4 +41,24 @@ public String filter(String urlString) throws URLFilterException {
     }
     return urlString;
   }
+/**Get a filter with the full classname if only it is activated through the nutchsite.xml*/
```

Comment:
This is messy Javadoc. You should make efforts to reference the configuration file properly e.g. nutch-site.xml as well as include the method parameter description(s).